src/coreclr/src/jit/lower.cpp

```diff
@@ -3233,16 +3270,30 @@ void Lowering::LowerRetStructLclVar(GenTreeUnOp* ret)
 // Arguments:
 //     call - The call node to lower.
 //
-// Note: it transforms the call's user.
+// Note: it skips multiReg return, it transforms the call's user for `GT_STOREIND`.
```

Comment:
I would separate this into two comments (they're not really related). I would also restate the first as "This handles only single-register returns."